### ISSUE TITLE
The Datalist documentation should not contain an unlabelled example

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4588]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/datalist

[WCAG 2.1 Guideline]
Success Criterion 4.1.2 Name, Role, Value
(Level A)
https://clarity.design/documentation/datalist

[OS]
any

[User Agents/Assistive Technologies]
any

[Component]
Datalist form input

[Description]
The Datalist documentation should not contain an unlabelled example

See the relevant HTML below.

&lt;div class=&quot;clr-input-group&quot;&gt;&lt;input clrdatalistinput=&quot;&quot; placeholder=&quot;No label&quot; name=&quot;Option&quot; aria-describedby=&quot;clr-form-control-18-helper&quot; id=&quot;clr-form-control-18&quot; list=&quot;clr-datalist-1&quot; class=&quot;clr-input ng-pristine ng-valid ng-touched&quot;&gt;&lt;!----&gt;&lt;datalist id=&quot;clr-datalist-1&quot;&gt;&lt;option value=&quot;Item1&quot; class=&quot;ng-star-inserted&quot;&gt;&lt;/option&gt;&lt;option value=&quot;Item2&quot; class=&quot;ng-star-inserted&quot;&gt;&lt;/option&gt;&lt;option value=&quot;Item3&quot; class=&quot;ng-star-inserted&quot;&gt;&lt;/option&gt;&lt;!----&gt;&lt;/datalist&gt;&lt;/div&gt;

[User Impact]
Clarity consumers will be encouraged to use this control without a label.

[Steps to Reproduce]
- Go to the Test URL
- Look at the &quot;no label&quot; example

[Recommendation]  
Add a visual and programmatic label to this example using the label element with the for attribute as seen below.

&lt;label for=&quot;clr-form-control-18&quot;&gt;Basic Example&lt;/label&gt;
&lt;input clrdatalistinput=&quot;&quot; placeholder=&quot;No label&quot; name=&quot;Option&quot; aria-describedby=&quot;clr-form-control-18-helper&quot; id=&quot;clr-form-control-18&quot; list=&quot;clr-datalist-1&quot; class=&quot;clr-input ng-pristine ng-valid ng-touched&quot;&gt;
